frb_example/pure_dart_multi/dart/lib/bridge_generated_api_1.dart

```diff
@@ -18,21 +18,26 @@ abstract class ApiClass1 {
   FlutterRustBridgeTaskConstMeta get kSimpleAdder1ConstMeta;
 }
 
-class ApiClass1Impl extends FlutterRustBridgeBase<ApiClass1Wire> implements ApiClass1 {
-  factory ApiClass1Impl(ffi.DynamicLibrary dylib) => ApiClass1Impl.raw(ApiClass1Wire(dylib));
+class ApiClass1Impl extends FlutterRustBridgeBase<ApiClass1Wire>
+    implements ApiClass1 {
+  factory ApiClass1Impl(ffi.DynamicLibrary dylib) =>
+      ApiClass1Impl.raw(ApiClass1Wire(dylib));
 
   ApiClass1Impl.raw(ApiClass1Wire inner) : super(inner);
 
-  Future<int> simpleAdder1({required int a, required int b, dynamic hint}) => executeNormal(FlutterRustBridgeTask(
-        callFfi: (port_) => inner.wire_simple_adder_1(port_, _api2wire_i32(a), _api2wire_i32(b)),
+  Future<int> simpleAdder1({required int a, required int b, dynamic hint}) =>
+      executeNormal(FlutterRustBridgeTask(
+        callFfi: (port_) => inner.wire_simple_adder_1(
+            port_, _api2wire_i32(a), _api2wire_i32(b)),
         parseSuccessData: _wire2api_i32,
-        parseErrorData: null,
+        parseErrorData: wire2apiPanicError,
```

Comment:
nit: Seems that it will yield *anyhow* errors, *in addition to* panic errors? Moreover, seems that we are trying to parse `Result` error (i.e. anyhow error in this specific case) here, not panic errors

Reply:
`wire2apiPanicError` is just for functions that return no `Result`, so the only error it can parse is the panic one. 

`pub fn simple_adder_1(a: i32, b: i32) -> i32`

returns no error

Reply:
I see. LGTM